node/tcp/src/protocols/handshake.rs

```diff
@@ -69,11 +69,11 @@ where
                             Ok(conn)
                         }
                         Ok(Err(e)) => {
-                            error!(parent: node.tcp().span(), "handshake with {} failed: {}", addr, e);
+                            debug!(parent: node.tcp().span(), "handshake with {} failed: {}", addr, e);
```

Comment:
This might trigger clippy warnings soon. Might be good to change it to this before merging.

```suggestion
                            debug!(parent: node.tcp().span(), "handshake with {addr} failed: {e}");
```